protocols/virtualfund/virtualfund.go

```diff
@@ -103,7 +103,7 @@ func New(
 	}
 
 	// Setup Ledger Channel Connections and expected guarantees
-	if myRole != 0 { // everyone other than Alice has a left-channel
+	if !init.isAlice() { // everyone other than Alice has a left-channel
```

Comment:
Is there any concern about referring to Alice and Bob in function names? We could use something like `isFirst/LastParticipant`